src/data/items/resourceXenobiology.js

```diff
@@ -0,0 +1,248 @@
+import { cloneDeep } from 'lodash';
+
+/*import SLIMES from "./slotCompanion";
+
+let slimeCoreEntries = Object.entries(SLIMES).filter(entry => { return entry[1].noCore != true; }).map(entry => {
+	let originalId = entry[0];
+	let originalItem = entry[1];
+
+	let newCore = cloneDeep(originalItem);
+    newCore.nocomplete = true; // don't count slime cores towards completion percentage
+	newCore.name = newCore.name + " Core";
+	newCore.healAmount = Math.round(newCore.healAmount * 1.3);
+	newCore.sellPrice *= 2;
+    newCore.equipmentSlot = "pocket";
+    newCore.fleeChance = null;
+    //if(originalItem.name.includes("Slime")){
+    //    let coreIcon = "@/assets/art/xenobio/cores/c_"+originalId.toString()+".png";
+    //    newCore.icon = require(coreIcon);
+    //}
+    newCore.icon = require("@/assets/art/xenobio/cores/c_slimeGray.png"); // TODO: Fix commented icons code above
+	return ["core_" + originalId, newCore];
+});
+
+const RESOURCE_XENOBIOLOGY = Object.assign(Object.fromEntries(slimeCoreEntries));
+
+// above is busted code to automatically copy cores from the slimes category in slotCompanion.js.
```

Comment:
I had written one (lengthy) idea in the discord, but if you wanted to do this the tedious way you could create a mapping from with somehting like

```
var slimeMapping = {
  "SlimeGray": require("@/assets/art/xenobio/cores/c_slimeGray.png"),
  "SlimeYellow": require("@/assets/art/xenobio/cores/c_slimeYellow.png"), 
}
```

This'd be kind of a pain in the ass tho